lib/elixir/lib/record.ex

```diff
@@ -298,6 +298,16 @@ defmodule Record do
   defp create(atom, fields, keyword, caller) do
     in_match = Macro.Env.in_match?(caller)
 
+    #backward compatiblity for Erlang, a :_ key sets all fields to a default value
+    {fields, keyword} = case Keyword.pop(keyword, :_) do
+      {nil, _} -> {fields, keyword}
+      {defaultValue, keyword2} -> 
+        fields2 = Enum.reduce fields, [], fn({k, _}, acc) ->
```

Comment:
`Enum.map/2` here would be fine: `Enum.map(fields, fn {k, _} -> {k, default_value} end)`.